### ISSUE TITLE
Is this page replacing the equivalent page on support.microsoft.com

### ISSUE THREAD

**Issue by JeremyTBradshaw** [MicrosoftDocs/OfficeDocs-Enterprise#133]
Hello,

I'm happy to see this page here on docs!  Just wondering if this page is replacing the longstanding [https://support.office.com/en-gb/article/office-365-urls-and-ip-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2](https://support.office.com/en-gb/article/office-365-urls-and-ip-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2) ?

It might be obvious, but I wanted to double-check so I can guide customers to the best destination for the information.

Thanks in advance.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 077406d8-bff1-496e-3ea2-b38558fd9355
* Version Independent ID: 57db3dec-2250-2dee-fd1a-ec6656911798
* Content: [Office 365 URLs and IP address ranges](https://docs.microsoft.com/en-us/office365/enterprise/urls-and-ip-address-ranges)
* Content Source: [Enterprise/urls-and-ip-address-ranges.md](https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/blob/live/Enterprise/urls-and-ip-address-ranges.md)
* Service: **o365-administration**
* GitHub Login: @JoeDavies-MSFT
* Microsoft Alias: **josephd**

**Comment by denisebmsft** [comment]
@JeremyTBradshaw thank you for the good question. We are looking into this right now.

**Comment by AndreaBarr** [comment]
@JeremyTBradshaw Thank you for this feedback

I want to make sure this replacement link gets addressed so I am making sure the Enterprise team gets this for investigation. 

I was able to get this into the backlog for the Enterprise team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
